src/graphql/resolvers/lookup-table.resolver.ts

```diff
@@ -0,0 +1,48 @@
+import { Logger } from '@map-colonies/js-logger';
+import { AxiosResponse } from 'axios';
+import { IConfig } from 'config';
+import { container } from 'tsyringe';
+import { Ctx, Query, Resolver } from 'type-graphql';
+import { Services } from '../../common/constants';
+import { IContext } from '../../common/interfaces';
+import { requestHandler } from '../../utils';
+import { LookupOption, LookupTablesData } from '../lookupTablesData';
+
+@Resolver()
+export class LookupTablesResolver {
+  private readonly logger: Logger;
+  private readonly config: IConfig;
+  private readonly serviceURL: string;
+
+  public constructor() {
+    this.logger = container.resolve(Services.LOGGER);
+    this.config = container.resolve(Services.CONFIG);
+    this.serviceURL = this.config.get('lookupTablesService.url');
+  }
+
+  @Query(() => LookupTablesData)
+  public async getLookupTablesData(@Ctx() ctx: IContext): Promise<LookupTablesData> {
+    try {
+      this.logger.info('[LookupTablesResolver][getLookupTablesData] fetching lookup tables data');
+
+      const data = await this.fetchLookupTablesData(ctx);
+
+      return data;
+    } catch (error) {
+      this.logger.error('[LookupTablesResolver][getLookupTablesData]', error as string);
+      throw error;
+    }
+  }
+
+  public async fetchLookupTablesData(ctx: IContext): Promise<LookupTablesData> {
+    const classificationPromise = requestHandler(`${this.serviceURL}/classification`, 'GET', {}, ctx);
```

Comment:
Cant we make it more generic somehow?
If not, consider this comment resolved :)